tests/test_project/templates/admin/base_site.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'admin/base_site.html' %}
+{% load static %}
+
+{% block extrastyle %}
+  {{ block.super }}
+  {% if OPENWISP_TEST_MODE %}
+    <link rel="stylesheet" type="text/css" href="{% static "admin/css/openwisp.css" %}" />
+    <link rel="icon" type="image/x-icon" href="{% static "ui/openwisp/images/favicon.png" %}" />
```

Comment:
Is it required to explicitly add this? 
Shouldn't this be already present from `{{ block.super }}`?

Reply:
During the test, those files are not loaded. That's why I added the new variable.

Reply:
These files are exactly the same, aren't they? 

https://github.com/openwisp/openwisp-utils/blob/9e182fd051e1b2142459014a9e5f6dc71a884ffa/tests/settings.py#L102-L120

Reply:
I printed it value during test and result was ``[{'href': '/static/custom-admin-theme.css', 'rel': 'stylesheet', 'type': 'text/css', 'media': 'all'}]``. It is getting manipulated in some place.

Reply:
@niteshsinha17 this test are flawed.

https://github.com/openwisp/openwisp-utils/blob/9e182fd051e1b2142459014a9e5f6dc71a884ffa/tests/test_project/tests/test_admin.py#L203-L242

I was't aware of `mocks` back when I started contributing. Can you please fix these tests?

Reply:
Thanks for the investigation. Fixed it.